src/VirtualAccount.php

```diff
@@ -21,13 +21,10 @@ public function all($options = array())
 
     public function close()
     {
-        $relativeUrl = $this->getEntityUrl() . $this->id;
+        $relativeUrl = $this->getEntityUrl() . $this->id . '/close';
 
-        $data = array(
-            'status' => 'closed'
-        );
 
-        return $this->request('PATCH', $relativeUrl, $data);
+        return $this->request('POST', $relativeUrl, $data);
```

Comment:
since $data is removed, it needs to be removed from here also, pls check if there is a default param for this. otherwise an empty array can be passed.

Also pls test the sdk before merging.

Reply:
removed $data